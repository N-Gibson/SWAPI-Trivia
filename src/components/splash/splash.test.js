import React from 'react';
import { shallow } from 'enzyme';
import Splash from './splash';

describe('Splash', () => {
  let wrapper;
  beforeEach(() => {
    let mockOrderRank = ['novice', 'intermediate', 'expert']
    wrapper = shallow(<Splash 
    orderRank={mockOrderRank}/>)
  })

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();    
  })
})