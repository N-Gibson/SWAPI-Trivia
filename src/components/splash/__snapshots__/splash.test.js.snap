// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Splash should match snapshot 1`] = `ShallowWrapper {}`;
